### ISSUE TITLE
can you please cut a new stable release? 

### ISSUE THREAD

**Issue by stefson** [kirides/GD3D11#172]
hey there, 

the latest stable dev9 release is from almost two years ago, can you please be so kind to cut a new release? You could for example tag an rc1, to allow everyone testing over the winter holidays, and finalize a few weeks later. 

thanks

**Comment by kirides** [comment]
i would love to, but there were a few issues in the recent past of the code and i currently work crunch-time at work and have about zero time to actually playtest and verify that nothing is broken.

I expect this to go way into the winter holidays and work getting less after new years.

**Comment by stefson** [comment]
yeah, I've kind of guessed this situation. the release candidate strategy would be beneficial here by allowing a wider audience to help with runtime testing.

**Comment by kirides** [comment]
Done, also merged a bunch of commits from other forks

**Comment by stefson** [comment]
yes I saw that :) 

mind you, what is it about the version you tagged with SK1 and SK2 respectively?

**Comment by kirides** [comment]
@SaiyansKing used these Tags/versions to tag his releases, probably to identify bug reports from players that use his releases.
